example/include/ranged_representation.h

```diff
@@ -50,15 +50,44 @@ class ranged_representation : public validated_type<T, is_in_range_t<T, Min, Max
   {
     return ranged_representation(-this->value());
   }
+
+  friend constexpr ranged_representation operator-(const ranged_representation& lhs, const ranged_representation& rhs)
+  {
+    return ranged_representation(lhs.value() - rhs.value());
+  }
+
```

Comment:
Yeah.

`-` now returns `ranged_representation`.
`+` still returns `T`.

When `-` returned `T`,
`ranged_representation` didn't satisfy `point_space`
because `+=` with a `T` value didn't work.

But if I remove the `-`,
and make the argument of `+=` and `-=` be a `T`,
it works.

Reply:
I am fine with either solution as long as it is consistent for the entire type. `+` and `-` should return the same type.